company-profiles/aim-india.md

```diff
@@ -8,7 +8,7 @@ Keeping in mind the most critical needs in today's context, we came up with weal
 
 ## Company size
 
-30000+ (as of January 2017)
+11-50 employees (as of October 2022)
```

Comment:
This is a drastic change in number, what's the source of ypur edit?

Reply:
Sir I have referred to their LinkedIn profile ["https://www.linkedin.com/company/aimindia/about/"](url)
Here is the screen shot of their page:-
![Screenshot aim](https://user-images.githubusercontent.com/114692575/194503000-35b1e9bd-36f5-4d54-81b6-b7423a153f0d.png)

Reply:
Thanks! Unsure where the original number comes from but that screenshot works for me.